Platform/CommonBoardPkg/CfgData/Template_BootOption.yaml

```diff
@@ -150,9 +150,9 @@ OS_TMPL: >
       value        : $(6)
   - SwPart_$(1)  :
       name         : Software Partition
-      type         : EditNum, DEC, (0,127)
+      type         : EditNum, DEC, (0,255)

```

Comment:
OS_TMPL is not used by anybody.   It was replaced by BOOT_OPTION_TMPL.
So let us remove all of OS_TMPL definitions from Tempate_BootOption.yaml to make it easier to maintain.